packages/block-editor/src/hooks/dimensions.js

```diff
@@ -58,6 +63,8 @@ export function DimensionsPanel( props ) {
 	const isDisabled = useIsDimensionsDisabled( props );
 	const isSupported = hasDimensionsSupport( props.name );
 	const spacingSizes = useSetting( 'spacing.spacingSizes' );
+	const paddingMouseOver = useVisualiserMouseOver();
+	const marginMouseOver = useVisualiserMouseOver();
```

Comment:
```suggestion
	const paddingMouseOver = useVisualizerMouseOver();
	const marginMouseOver = useVisualizerMouseOver();
```

Switch to US spelling.

Reply:
The King'z english has gone 😄